### ISSUE TITLE
Reconcile Test Data with Database Schema

### ISSUE THREAD

**Issue by edward-hong** [PaperHat/centripetal#62]
Depends on #77

Some of the test data doesn't have corresponding columns in the database tables. Possibly add columns to existing tables or add new tables to include extra information.

Here's a table detailing the data values that don't have a place to be put in Postgres: https://docs.google.com/spreadsheets/d/1Z4E9Bfp4M8CfIED7saZXZpsL6RRb8f7hpJD478qiX8Y/edit#gid=0

**Comment by stanley-roache** [comment]
@edward-hong would you mind putting up that info collected on how and where the schema differs from test data? We could split this up into chunks

**Comment by stanley-roache** [comment]
Necessary short term for test data insert (on this ticket):
- add nullable job_title column to account table
- create presentation_type table

Future alterations (not on this ticket but important later):
- conference role tables e.g. panelist. 
We talked about adding tables that describe a relationship between an account holder and a specific event (user X is a ' Panelist' at event Y), the table will also contain information specific to that relationship e.g. a select or tailored bio of the panelist showing their relevance to the event
